RELEASE.md

```diff
@@ -1,6 +1,13 @@
-# Release 1.0.4
+# Release 1.0.6
```

Comment:
The version number is ahead of the other two `1.0.6`, I assume it is correct

Reply:
I am also a bit confused because this one is 1.0.6 and the others are 1.0.5

Reply:
@changanxian @eri9600  yes, it is correct. `maps-app-android` has had an additional patch release where its minimum API level was updated to 21. The other two apps were already at that level.

Reply:
https://github.com/Esri/maps-app-android/releases
 
https://github.com/Esri/mapbook-android/releases

https://github.com/Esri/nearby-android/releases

Reply:
ok